indico/cli/user.py

```diff
@@ -217,6 +219,42 @@ def unblock(user_id):
         click.secho('Successfully unblocked user', fg='green')
 
 
+@cli.command()
+@click.argument('user_id', type=int)
+def anonymize(user_id):
+    """Anonymize a given user.
+
+    This will remove all personal data about the given user including
+    their name, affiliation and emails. Note that any data tied to events
+    such registrations will not be deleted.
+    """
+    user = User.get(user_id)
+    if user is None:
+        click.secho('This user does not exist', fg='red')
+        return
+    _print_user_info(user)
+    if click.confirm(click.style('Anonymize this user?', fg='yellow')):
```

Comment:
`if not ...: return`

and I'd move all the logic to a util in indico/module/users